src/core/thread/network_data_leader_ftd.cpp

```diff
@@ -1003,14 +1003,25 @@ void Leader::FreeContextId(uint8_t aContextId)
 
 void Leader::StartContextReuseTimer(uint8_t aContextId)
 {
+    // Start the reuse timer for `aContextId` if it is not already
+    // scheduled.
+
+    VerifyOrExit(mContextLastUsed[aContextId - kMinContextId].GetValue() == 0);
```

Comment:
Thanks for the fix @abtink. I see mContextLastUsed is memset to 0 when the device becomes a leader. 
The fix looks good, just wanted to confirm, this guarantees that in all scenarios, mContextLastUsed value timestamp is updated only once to GetNow()?

StopContextReuseTimer could also set mContextLastUsed value back to 0 (Invoked from AddBorderRouter and HandleNetworkDataRestoredAfterReset). The mContextLastUsed reset from HandleNetworkDataRestoredAfterReset is a different scenario (#8854) but I believe AddBorderRouter call reseting mContextLastUsed is an expected scenario to reset (the prefix becoming active again)?

Reply:
> StopContextReuseTimer could also set mContextLastUsed value back to 0 (Invoked from AddBorderRouter and HandleNetworkDataRestoredAfterReset). The mContextLastUsed reset from HandleNetworkDataRestoredAfterReset is a different scenario (https://github.com/openthread/openthread/issues/8854) but I believe AddBorderRouter call reseting mContextLastUsed is an expected scenario to reset (the prefix becoming active again)?

Yes, we want to stop the last used timer when the prefix is in use again.